index.js

```diff
@@ -417,15 +417,9 @@ module.exports = class Minipass extends Stream {
       this[EMITTED_ERROR] = data
     }
 
-    // TODO: replace with a spread operator when Node v4 support drops
-    const args = new Array(arguments.length)
-    args[0] = ev
-    args[1] = data
-    if (arguments.length > 2) {
-      for (let i = 2; i < arguments.length; i++) {
-        args[i] = arguments[i]
-      }
-    }
+    const args = [...arguments];
+    args[0] = ev;
+    args[1] = data;
 
     try {
       return super.emit.apply(this, args)
```

Comment:
Can get rid of all of the above, I think, and just change this one line to:

```
return super.emit(...arguments)
```

That's what the `TODO` item was saying.  `arguments[0]` and `arguments[1]` are already ev and data, respectively.